### ISSUE TITLE
Preserve Newlines and Output Scrubbing

### ISSUE THREAD

**Issue by AustinTSchaffer** [smallwat3r/shhh#7]
I actually used this app today to share credentials with some colleagues. Ngrok is super cool too.

Anyway, I noticed that somewhere along the data's round-trip, newlines are removed. This is a little annoying, since I like sending credentials in this format:

```
Brief Title of Secret Message

App URL: http://something.somewhere
Username: asdf
Password: reallySecureP@ssword
```

Debugging the application shows that the newlines are being dropped as part of the display of the secret message, and that the new line characters are available after decrypting the message. There is no way to get it to display newlines as a user (unless you use `<br/>` tags), regardless of how many new lines you use in between each line of the message.

**Comment by AustinTSchaffer** [comment]
Something else I noticed while thinking about using HTML tags as part of a secret message, is that the app does not protect against Cross Site Scripting (XSS) attacks.

If you make a secret with the content `<script>alert("test")</script>`, the script element is evaluated and an alert window is displayed.

**Comment by smallwat3r** [comment]
Thanks @AustinTSchaffer - Indeed I will have a look at this

**Comment by smallwat3r** [comment]
Fixed on #10
